legislation_analysis/processing/ner.py

```diff
@@ -59,7 +59,7 @@ def apply_ner(text: str, ner_lst: list) -> list:
         return ner_lst
 
     @staticmethod
-    def post_process_ner(ner: list) -> list:
+    def post_process(ner: list) -> list:
```

Comment:
If it's only being called internally, you could throw a change it to `__post_process` so that it's known as a `private` function: https://www.geeksforgeeks.org/private-methods-in-python/

I always thought it was one _, but it seems like it's two. I'll update my functions in the next PR.